content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,144 @@
+
+## Proof of Stake
+
+Then came the [Proof of Stake](https://www.investopedia.com/terms/p/proof-stake-pos.asp) (PoS) consensus algorithms (e.g. Casper).
+This type no longer needs the computational power used for blocks validating but rather an amount staked by network participants (being single or in clusters).
+Staking is putting some native cryptocurrency as a collateral in order to become a validating entity in the network.
+The higher the stake, the higher the chances of getting to write a block on behalf of the whole ecosystem and in return receive some cryptocurrency.
+Two keywords are different here compared to PoW: (1) Computers that participate in PoS consensus are called Validators, (2) Once a block has been created and accepted by the network, it is said to be forged (not mined).
+Then the validator gets some reward.
+If a user does not want to run his own validator, then he can generally simply "delegate" his tokens to the validators, which is like staking without running his own validator, and add the weight of the delegated tokens to the staked tokens of the chosen validator, in return for giving part of the reward to the validator.
+If a validator tries to be malicious then it gets penalised losing some of its staked cryptocurrency.
+
+One key difference between PoS and PoW is that a block on a PoS chain has deterministic finality, meaning that once it has been accepted fully, there is no way to ever undo it.
+In Pow, it can (in theory) always be undone by creating a longer valid chain of blocks.
+PoS is relatively faster than PoW as there is no need to solve complex cryptographic puzzles to get to be the block writer and the process is much more streamlined.
+Also no unnecessary wastage of energy means PoS is more environment friendly.
+PoS algorithms based on BFT need 2/3+1 of the nodes to be honest, so the required threshold is higher than PoW where an attacker needs 51% of all computing power.
+Sustainability, security are achieved in a PoS model like the one in Cardano.
+In PoS, explicit agreement is required before a block becomes valid.
+If a sufficient number of validators do not sign the block then it will be rejected even if all consensus rule is followed by the block.
+Only after the necessary number of participants to the consensus algorithm explicitly vote for the block by signing it, will it become valid.
+This means that a lot of communication is needed.
+In general, voting on the next correct block requires multiple rounds of communication (preparation round, confirmation round and commit round), and the more validators there are, the more overhead there is for everyone to get their messages to the leader of a round.
+Due to this high number of communication scalability is hindered.
+
+Also the higher the adaptation, the higher the total value represented by the native cryptocurrency and therefore higher the value that needs to be locked up as staking by the nodes.
```

Comment:
I'm confused by this - "the higher the adaption"? What adaptation is this?

Reply:
Adaptation by community in general when the particular cryptocurrency gets more popular.

Reply:
Ah, do you mean adoption?